mono/mini/aot-compiler.c

```diff
@@ -8524,6 +8627,7 @@ emit_code (MonoAotCompile *acfg)
 	 * To work around linker issues, we emit a table of branches, and disassemble them at runtime.
 	 * This is PIE code, and the linker can update it if needed.
 	 */
+	
```

Comment:
We usually try to avoid whitespace-only changes like this since it creates noise in teh git blame.